examples/Tutorials/modflow6plugin/tutorial02_mf6_plugin.py

```diff
@@ -0,0 +1,147 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.5.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# # FloPy plugin Tutorial 2: Creating a FloPy plugin
+#
+# This tutorial demonstrates how to use the FloPy plugin template generator to
+# get you started on your own FloPy plugin.  For more information see the
+# notebooks in the flopy distribution's examples/Notebooks/dev folder.
+
+# ## Getting Started
+#
+# MODFLOW-6 supports a BMI interface which allows you to modify its behavior.
+# Support for accessing the BMI interface using python is provided through the
+# modflowapi library.  FloPy plugins are FloPy-integrated components that
+# modify MODFLOW-6 functionality using the modflowapi library.  The FloPy
+# plugins architecture streamlines the process of modifying MODFLOW-6
+# functionality by:
+#
+# * Providing template generators to help rapidly set up your plugin's
+# interface
+# * Automatically loading, saving, and running FloPy plugins as part of a
+# MODFLOW-6 simulation
+# * Providing an interface that allows multiple FloPy plugins to run together
+#
+# This tutorial describes the process of generating your own custom FloPy
+# plugin template using the flopy template generator.  The plugin template
+# includes all files you will need to create your basic FloPy plugin.
+
+import flopy
+
+# To create a plugin template, you first need to decide which type of model
+# your plugin will reside in.  Define the model type with its three letter
+# abbreviation.
+
+# define the model type
+mt = "gwf"
+
+# Next, pick a unique three letter abbreviation for your plugin.  This
+# abbreviation should be different from any MODFLOW-6 package or other flopy
+# plugin abbreviation.
+
+# define the stress plugin three letter abbreviation
```

Comment:
Here you use abbreviation and name, which is different from earlier descriptions.

Reply:
I change “extension” to “abbreviation” everyone for consistency.